core/http-auth-event-stream/pom.xml

```diff
@@ -28,7 +28,7 @@
     <artifactId>http-auth-event-stream</artifactId>
     <name>AWS Java SDK :: HTTP Auth Event Stream</name>
     <description>
-        The AWS SDK for Java - HTTP Auth module contains interfaces and implementations for http authentication
+        The AWS SDK for Java - HTTP Auth module contains interfaces and implementations for generic HTTP authentication
```

Comment:
Maybe:
The AWS SDK for Java - HTTP Auth Event Stream module contains interfaces and implementations for authentication of event streams in HTTP services.